cfgov/v1/handlers/blocks/feedback.py

```diff
@@ -97,7 +97,7 @@ def success(self):
                      'message': "Be sure to also sign up for our email list "
                      "to get our blog posts and other tips about homebuying "
                      "and mortgages in your inbox. We'll also let you know "
-                     "when we make updates to Owning a Home."}
+                     "when we make updates to Buying a House."}
```

Comment:
@higs4281 I presume this is a safe change, but how is this tested?

Reply:
👍 Looks safe to me. 
The feedback block is pretty [well-tested](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/tests/handlers/blocks/test_feedback.py), but we don't test the specific success language for suggestion responses.